### PR TITLE
.DAE Import Error Messages

### PR DESCRIPTION
**Pull request by ssbucarlos**
Adds error messages to two common import issues with .DAE model import. (Missing Armature + Invalid Materials)

### CODE REVIEW

File: StudioSB/MainForm.cs [hunk 1]
```diff
@@ -720,7 +720,19 @@ public static SBScene LoadScene(string filePath, SBScene loadedScene)
...
             {
-                var ioModel = IONET.IOManager.LoadScene(filePath, settings);
+                IONET.Core.IOScene ioModel;
+                try
+                {
+                    ioModel = IONET.IOManager.LoadScene(filePath, settings);
+                }
+                catch (ArgumentNullException e)
+                {
+                    throw new ArgumentNullException("This error is usually caused by missing textures. Go to blender, change shading mode to 'Material Preview', and if ur model looks pink, its missing textures.\n Original Error Msg: \n" + e.Message, e);
```
Comment (ScanMountGoat) [inline StudioSB/MainForm.cs:730]: This can just contain the first sentence to avoid confusing non Blender users.

File: StudioSB/MainForm.cs [hunk 2]
```diff
@@ -720,7 +720,19 @@ public static SBScene LoadScene(string filePath, SBScene loadedScene)
...
+                }
+                catch (NullReferenceException e)
+                {
+                    throw new NullReferenceException("This error is usually caused by a not exporting an Armature along with the Meshes. \nOriginal Message: \n" + e.Message , e);
```
Comment (ScanMountGoat) [inline StudioSB/MainForm.cs:734]: I would prefer "skeleton", "rigging data", etc. I've only ever heard "armature" used with Blender.

**Comment by ScanMountGoat** [comment]
Thanks for looking into this. I would like the error messages to be more general and not specific to Blender, and then I'll go ahead and merge this.

**Comment by ssbucarlos** [comment]
ah ok, i was under the impression that max users exported with FBX and didn't run into these issues. Ill go ahead and make the error messages more general then

**Pull request merged**